Makefile

```diff
@@ -23,13 +23,15 @@ test:
 	go install -v gopkg.in/reform.v1/internal/test/models
 	go test -i -v
 	go install -v github.com/kisielk/errcheck \
-					github.com/golang/lint/golint \
+					github.com/alecthomas/gometalinter \
 					github.com/AlekSi/goveralls
+	gometalinter --install
 
 check: test
 	go vet ./...
 	-errcheck ./...
 	golint ./...
+	gometalinter ./...
```

Comment:
Some check failures should abort build, some not. For example, `go vet` checks errors be fatal, `golint` not. Can we tweak it here? (I don't know if we can.)